sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusSenderOptions.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+
+namespace Azure.Messaging.ServiceBus
+{
+    /// <summary>
+    /// The set of options that can be specified when creating a <see cref="ServiceBusSender"/>
+    /// to configure its behavior.
+    /// </summary>
+    public class ServiceBusSenderOptions
+    {
+        /// <summary>
+        /// A property used to set the <see cref="ServiceBusSender"/> ID to identify the client. This can be used to correlate logs
```

Comment:
nit:  We should consider updating this block to remove the implied guarantee about uniqueness, since we don't have control over the value in all cases.